README.md

```diff
@@ -154,12 +154,19 @@ The following steps allow you to create a modified copy of one of the standard O
 7. [Optional] Replace the PNG image files in the `slides` and `slides_vga` directory with your own custom installer slides
 
 8. Edit the following fields in the `partitions.json` file contained in the folder that you just created
-  1. "partition_size_nominal" - replace the numerical value with the size of the paritions in your custom OS version
-  2. "uncompressed_tarball_size" - replace the numerical value with the size of your filesystem tarballs when uncompressed
-
-9. Replace the `.tar.xz` root and boot filesystem tarballs with copies created from your custom OS version (these instructions assume you're only using a single OS at a time with NOOBS - they won't work if you're running multiple OSes from a single SD card). The name of these tarballs needs to match the labels given in `partitions.json`.
-  1. To create the root tarball you will need to run `tar -cvpf <label>.tar /* --exclude=proc/* --exclude=sys/* --exclude=dev/pts/*` from within the root filesystem of your custom OS version. You should then compress the resulting tarball with `xz -9 -e <label>.tar`.
-  2. To create the boot tarball you will need to run `tar -cvpf <label>.tar .` at the root directory of the boot partition of your custom OS version. You should then compress the resulting tarball with `xz -9 -e <label>.tar`.
+  1. "partition_size_nominal" - replace the numerical value with the size of the paritions (round up) you want in your custom OS version. (Also note the size is in MB).
+  2. "uncompressed_tarball_size" - replace the numerical value with the size of your filesystem tarballs (round up) when uncompressed (this can be obtained by running the command ll within the directory where the tarball is located). Also note the size is in MB.
```

Comment:
The actual size should be in MiB (1024x1024 bytes)

Reply:
@procount are you sure about that?

I wise man (@lurch) once told me this:

> * partitions.json needs to have "correct" partition sizes (the base-10
version rather than the base-2 version, because that's what parted
uses) for each partition
>  - the "uncompressed_tarball_size" should be the size in MB, rounded
up to the next whole MB (i.e. the uncompressed root.tar file has a
size of 2,336,133,120 bytes and so should have
"uncompressed_tarball_size": 2337). This is simply used for the
install-progress-bar.
>  - the "partition_size_nominal" is the minimal size that the
partition will be created at. For Raspbian's root partition, this
experimentally needs to be approximately 400MB larger than the
"uncompressed_tarball_size" as this then leaves (at least) enough
space for the 100MB swapfile that gets created during the first
bootup, and to allow the user to run both apt-get upgrade and
rpi-update before running out of diskspace.